docs/src/content/docs/ops/azure.adoc

```diff
@@ -11,10 +11,91 @@ NOTE: For more details on getting started with Azure, see the link:guides/starti
 In order to run an Azure based XTDB cluster, the following infrastructure is required:
 
 * An **Azure Storage Account**, containing a **Storage Account Container**.
-* A user assigned managed identity for authentication with Azure services.
+* A **User Assigned Managed Identity** for authentication with Azure services.
 * A **Kafka cluster** for transaction logs.
 ** For more information on setting up Kafka for usage with XTDB, see the link:config/tx-log/kafka[Kafka configuration^] docs.
 * XTDB nodes configured to communicate with the Kafka cluster and Azure Storage Account/Container.
+* (On Kubernetes) A **Federated Identity Credential** setup for the desired Kubernetes Namespace/Service account to give access to the **User Assigned Managed Identity**.
+
+'''
+
+[#terraform]
+== Terraform Templates
+
+To set up a basic version of the required infrastructure, we provide a set of Terraform templates specifically designed for Azure.
+
+These can be fetched from the XTDB repository using the following command:
+
+```bash
+terraform init -from-module github.com/xtdb/xtdb.git//terraform/azure
```

Comment:
just to double-check: `--from-module`?

Reply:
https://developer.hashicorp.com/terraform/cli/commands/init#copy-a-source-module For whatever reason, it actually is `-from-module`